lib/keycloak-api-rails/helper.rb

```diff
@@ -83,6 +83,7 @@ def self.current_user_roles(env)
     end
 
     def self.read_token_from_query_string(uri)
+      return "" unless uri.present?
```

Comment:
The rest of the code does not "break" methods with `return`. To make it consistent, could you replace this instruction with something like:
```ruby
if uri.present?
 ...
else
  ""
end
```